clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/ops/CopyLastGoogleServerGroupAtomicOperation.groovy

```diff
@@ -84,37 +87,33 @@ class CopyLastGoogleServerGroupAtomicOperation implements AtomicOperation<Deploy
     task.updateStatus BASE_PHASE, "Initializing copy of server group $description.source.serverGroupName..."
 
     // Locate the ancestor server group.
-    // TODO(duftler): Replace this with a call to GoogleClusterProvider.
-    InstanceGroupManager ancestorServerGroup = GCEUtil.queryManagedInstanceGroupInRegion(description.credentials.project,
-                                                                                         description.source.region,
-                                                                                         description.source.serverGroupName,
-                                                                                         description.credentials)
+    def ancestorServerGroup = GCEUtil.queryServerGroup(googleClusterProvider,
```

Comment:
Something to keep in mind: if clone server group ever becomes a pipeline stage - it will need to be preceded by a force cache refresh of the server group being cloned